negotiation/neural/generator.py

```diff
@@ -113,7 +123,11 @@ def unbottle(m):
         # (2) Repeat src objects `beam_size` times.
         #src_map = rvar(batch.src_map.data) \
         #    if data_type == 'text' and self.copy_attn else None
-        memory_bank = rvar(memory_bank.data)
+        if batch.num_context > 1:
+            memory_bank[0] = rvar(memory_bank[0].data)
+            memory_bank[1] = rvar(memory_bank[1].transpose(0,1).data).transpose(0,1)
```

Comment:
why do you do 2 transpose here

Reply:
also can you add a comment about what's in the list memory_bank and the dim of each element

Reply:
see comment in Batch.